active/0000-nested-addon-imports.md

```diff
@@ -0,0 +1,63 @@
+- Start Date: (fill me in with today's date, 2016-08-06)
+- RFC PR: 
+- ember-cli Issue: 
+
+# Summary
+
+The new asset import API, available since ember-cli 2.7.0, offers the possibility to 
+import Ember CLI addons from Ember CLI addons (aka, nested addons).
+This RFC intends to find solutions for typical edge cases.
+
+# Motivation
+
+Since ember-cli 2.7.0, addons can be nested if the nesting floows the following
```

Comment:
Nested addons has been possible for a _long_ time. `this.import` just makes it easier to do the right thing.